plugins/BEdita/Core/config/Migrations/20240626123658_UpdateProperties.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+
+class UpdateProperties extends AbstractMigration
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function up(): void
+    {
+        $this->table('properties')
+            ->addColumn('default_value', 'text', [
+                'after' => 'read_only',
+                'comment' => 'Default value',
+                'length' => null,
+                'null' => true,
+            ])
+            ->addColumn('property_options', 'text', [
```

Comment:
why not setting as a json column?

```suggestion
            ->addColumn('property_options', 'json', [
```

and add `->setColumnType('property_options', 'json')` in `PropertiesTable::getSchema()`. You can then json-encoding in `Property` utility and set the `property_options` property as `array` in the entity